### ISSUE TITLE
Highlight comments in *.feature files in green

### ISSUE THREAD

**Issue by MiladSadinam** [cucumber/vscode#249]
### 🤔 What's the problem you're trying to solve?

We use comments to structure our scenarios. Unfortunately, they have no highlighting at all and are rather missed.

### ✨ What's your proposed solution?

Highlight them in green as it is usual in almost any editor.

### ⛏ Have you considered any alternatives or workarounds?

I tried to install other Gherkin highlighting extensions in addition to this extension, but then the highlighting of "undefined" steps would not work anymore.

I also tried to use the "Token inspector" on the comments to check if I can manually highlight them, but the inspector says `No grammar or semantic tokens available.` for comments.

**Comment by VasilySazonenko** [comment]
I agree with author. I desperately need any semantic token on comment lines to make them different color from active gherkin lines!
